frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/upgrade.scenario.ts

```diff
@@ -0,0 +1,43 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import {
+  changeChannel,
+  changeCatalogSourceImage,
+  channelChangeButton,
+  channel43,
+  channel44,
+  storageClusterVersion,
+  waitUntilStorageClusterReady,
+} from '../../views/upgrade.view';
+import { CHANNEL_43, CHANNEL_44 } from '../../utils/consts';
+
+describe('Test OCS version upgrade.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
+  });
+
+  xit('Test channel change from 4.2 to 4.3', async () => {
+    // Pre-requisites: OCS 4.2 is installed
+    await changeCatalogSourceImage('latest-stable-4.3.0');
+    await changeChannel(channel43);
+    expect(channelChangeButton.getText()).toEqual(CHANNEL_43);
+  });
```

Comment:
Do we require this test?

Reply:
Well, for a very long time this was the only test actually working, before 4.3 to 4.4 upgrade was fixed. I think we can leave it here for now, maybe get rid of it later.